examples/structs/src/main.sw

```diff
@@ -84,5 +81,5 @@ fn struct_destructuring() {
         Point { x: 2, y: 4 },
         Point { x: 3, y: 6 },
     );
-    let (Point{x:x0, y:y0, }, Point{x:x1, y:y1, }) = struct_in_tuple;
+    let (Point{x:x0, y:y0}, Point{x:x1, y:y1}) = struct_in_tuple;
```

Comment:
How come the `Point` on lines 81 & 82 is formatted as `Point<space>{` but on line 84 there is no space?

Reply:
this has to do with how inner expressions are formatted within the tuple expr type, I've meant to file an issue about this